torch/testing/_internal/common_quantized.py

```diff
@@ -21,16 +21,27 @@ def _conv_output_shape(input_size, kernel_size, padding, stride, dilation,
                      * (dilation - 1)) / stride) + 2 * output_padding + 1
 
 # Quantization references
-def _quantize(x, scale, zero_point, qmin=None, qmax=None, dtype=np.uint8):
+def _quantize(x, scale=None, zero_point=None, qmin=None, qmax=None,
+              dtype=np.uint8):
```

Comment:
Might be a good idea to update the docstring for this function to reflect the new behavior.

Reply:
Do you mean the comment here? This is just an internal function for testing -- do we have a docstring?